pkgs/top-level/python-packages.nix

```diff
@@ -4243,20 +4243,23 @@ in {
 
   jaxlib-bin = callPackage ../development/python-modules/jaxlib/bin.nix {
     cudaSupport = pkgs.config.cudaSupport or false;
-    inherit (self.tensorflow) cudaPackages;
```

Comment:
Why was it using tensorflow's cuda first and no longer?

Reply:
This was actually at my request way back when @ndl first introduced the jaxlib source build. The original thinking being that people may be using JAX and TF together in which case it would be inefficient to have two different CUDA versions lying around. As time has gone on, it's turned out to be more of a burden on JAX than a benefit IMHO. TF just has some weird and outdated CUDA version constraints that are a PITA to work with (currently stuck on <=11.2). 

Upgrading the CUDA version was essential to fix the build in this case since we needed to use a CUDA version with support for gcc11.